logic/scorer/scorer_test.go

```diff
@@ -10,6 +11,295 @@ import (
 	"github.com/joshprzybyszewski/cribbage/model"
 )
 
+func BenchmarkHandPoints(b *testing.B) {
+	hand := make([]model.Card, 0, 5)
+	seen := make(map[model.Card]struct{}, 5)
+	for len(hand) < 5 {
+		c := model.NewCardFromNumber(rand.Intn(52))
+		if _, ok := seen[c]; ok {
+			continue
+		}
+		hand = append(hand, c)
+		seen[c] = struct{}{}
+	}
+
+	b.Run(`scoring random hand`, func(b *testing.B) {
+		for i := 0; i < b.N; i++ {
+			s := HandPoints(hand[0], hand[1:])
+			require.Less(b, s, 30)
+			require.GreaterOrEqual(b, s, 0)
+		}
+	})
+}
+
+func TestScoreRuns(t *testing.T) {
+	tests := []struct {
+		desc      string
+		hand      string
+		expType   scoreType
+		expPoints int
+	}{{
```

Comment:
:cactus: are these `testCases` copy/pasted from elsewhere? Would it be beneficial to pull them out somewhere for common usage? :noidea:

Reply:
could be helpful to have a `testdata` file that defines a lot of hands with expected total points, run points, etc.

then again, this whole test probably isn't _super_ useful anymore because if we can't score runs our ScoreHand tests would fail :shrug: